### DISCUSSION TITLE
Little problem

### DISCUSSION THREAD

**Discussion by Saw1327** [spartacus04/jext-reborn#322] [Q&A] [answered]
So i have server on version 1.20.1, i'm from Ukraine and i can't understand what exactly password is requesting from me and where i can see it.If it plugin web password, i change him but nothing happaned.And if i try do it without web gui login, i have some bug that tell's me some functions can't work and texture pack gots disabled after activation.
![Screenshot_1](https://github.com/spartacus04/jext-reborn/assets/167926540/10413c7b-0589-4384-898f-2153c385f5f7)

**Comment by spartacus04** [comment]
You can find the password in the `config.json` file inside the plugin directory. By default it's an empty string, so just clicking submit should work

**Reply by Saw1327** [reply]
When i clicking submit it just doing this.Can it be because i use Opera GX or it can be another problem?
![Screenshot_2](https://github.com/spartacus04/jext-reborn/assets/167926540/496839fd-2b62-4d01-bd0f-a26cf7e9b7e4)

**Reply by spartacus04** [reply, marked as answer]
Since opera GX is chromium based you need to enable mixed content. You can use this tutorial [here](https://experienceleague.adobe.com/en/docs/target/using/experiences/vec/troubleshoot-composer/mixed-content), it doesn't include a tutorial specifically for opera GX, however I suppose it should be really similar to other browsers.

If it doesn't work yet you can download the desktop app from the "See details" button and use that

**Reply by Saw1327** [reply]
So thanks but i found my own solution, i just install texture pack inside plugin and also install on my own minecraft.Thank you for your time and gratiest plugin. <3
